admin/acceptance.py

```diff
@@ -195,6 +203,23 @@ def ensure_keys(reactor):
             found.  Otherwise, fails with ``AgentNotFound`` or ``KeyNotFound``.
         """
 
+    def extend_cluster(reactor, cluster, count, tag, starting_index):
+        """
+        Create new nodes and add them to the given cluster.
+
+        :param reactor: Reactor to use.
```

Comment:
The reactor is the core Twisted event loop runner. Hence why it is pervasive through the code, and generally not documented everywhere it appears.  If it is typed, it is usually as `twisted.internet.interfaces.IReactorTime` which is the interface shared by Twisted reactors and the `twisted.internet.task.Clock` used for testing.